src/igvfd/audit/measurement_set.py

```diff
@@ -211,24 +212,43 @@ def audit_missing_institutional_certification(value, system):
 
 
 @audit_checker('MeasurementSet', frame='embedded')
-def audit_CRISPR_screen_lacking_auxiliary_set(value, system):
+def audit_CRISPR_screen_missing_auxiliary_set(value, system):
     '''
     [
         {
             "audit_description": "Measurement sets from CRISPR-based assays are expected to link to a gRNA sequencing auxiliary set.",
             "audit_category": "missing auxiliary set",
             "audit_level": "NOT_COMPLIANT"
-        },
+        }
+    ]
+    '''
+    description = get_audit_description(audit_CRISPR_screen_missing_auxiliary_set)
+    if value['assay_term']['term_name'] in ['proliferation CRISPR screen',
```

Comment:
should not we include variant flowfish here?

Reply:
Variant FlowFISH doesn't have gRNA sequencing